src/AzureAuth/CommandMain.cs

```diff
@@ -159,6 +159,12 @@ public CommandMain(CommandExecuteEventData eventData, ILogger<CommandMain> logge
         [Option(ConfigOption, "The path to a configuration file.", CommandOptionType.SingleValue)]
         public string ConfigFilePath { get; set; }
 
+        /// <summary>
+        /// Gets or sets the cache file name.
+        /// </summary>
+        [Option(CacheOption, "Override the default cache file location.", CommandOptionType.SingleValue)]

```

Comment:
Given the cache file is Windows Only, but we need the default value to compile still for Mac. I think we can conditionally set the option to hide this flag from the help text on Mac. CommandLineUtils has a way to do this, and we have a build constant you can use to set a bool if on windows, or not.